src/main/java/com/emc/ecs/nfsclient/nfs/NfsSetAttrRequest.java

```diff
@@ -80,8 +83,7 @@ public void marshalling(Xdr xdr) {
             xdr.putBoolean(false);
         } else {
             xdr.putBoolean(true);
-            xdr.putUnsignedInt(_guardTime.getSeconds());
-            xdr.putUnsignedInt(_guardTime.getNanoseconds());
+            _guardTime.marshalling(xdr);
```

Comment:
Doesn't this just undo the commit (https://github.com/EMCECS/nfs-client-java/pull/25/commits/8ad3495d406f7ee273b675217f0663ce7f450f39) before it?

Reply:
No - it keeps the marshaling of the boolean, and marshaling of the other two fields (seconds and nanoseconds) is restored to being done by the NfsTime class, so that they don't need to be exposed externally.

Reply:
Okay, I see now.